src/data-interfaces/HposInterface.js

```diff
@@ -25,14 +17,17 @@ const axiosConfig = {
   }
 }
 
-export function hposCall ({ method = 'get', path, authToken, apiVersion = 'v1', headers: userHeaders = {} }) {
+export function hposCall ({ method = 'get', path, apiVersion = 'v1', headers: userHeaders = {} }) {
   if (MOCK_HPOS_CONNECTION) {
     return mockCallHpos(method, apiVersion, path)
   } else {
     return async params => {
-      const fullPath = process.env.REACT_APP_HPOS_URL + '/' + apiVersion + '/' + path
+      // For Robbie: this url should be probably constructed from window.location, right?
```

Comment:
having it be an environment variable gives us more control in developing. Currently
`'https://' + window.location.hostname + '/' + apiVersion + '/' + path`. This change will work once we have HPOS as part of our dev environment.

Reply:
👍